wallet.Dockerfile

```diff
@@ -4,6 +4,7 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 COPY target/release/wallet-api /app/wallet-api
+COPY target/release/wallet-cli /app/wallet-cli
```

Comment:
I think we also need `RUN chmod +x /app/wallet-cli` (see below where we do it for `wallet-api`)